roles/ssh_jumper/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Make sure ~/.ssh/cifw_ssh_config.d directory exists
+  delegate_to: "{{ cifmw_ssh_jumper_config.target | default(inventory_hostname) }}"
+  ansible.builtin.file:
+    path: "{{ cifmw_ssh_jumper_config.ssh_dir | default((ansible_user_dir, '/.ssh') | path_join) }}/cifw_ssh_config.d"
+    state: directory
+    mode: '0700'
+
+- name: Include ~/.ssh/config.d/*.conf
+  delegate_to: "{{ cifmw_ssh_jumper_config.target | default(inventory_hostname) }}"
+  ansible.builtin.lineinfile:
+    insertbefore: BOF
+    create: true
+    state: present
+    path: "{{ cifmw_ssh_jumper_config.ssh_dir | default((ansible_user_dir, '/.ssh') | path_join) }}/config"
+    mode: '0600'
+    line: 'Include cifw_ssh_config.d/*.conf'
+
+- name: "Inject ssh jumpers for {{ cifmw_ssh_jumper_config.hostname }}"
+  delegate_to: "{{ cifmw_ssh_jumper_config.target | default(inventory_hostname) }}"
+  ansible.builtin.template:
+    src: ssh_host.conf.j2
+    dest: "{{ cifmw_ssh_jumper_config.ssh_dir }}/cifw_ssh_config.d/cifmw-{{ cifmw_ssh_jumper_config.hostname }}.conf"
```

Comment:
this will explode mid-air if someone has multiple lab in parallel.
You want to take the ProxyJump host name into account, or the `inventory_hostname` to build that file name

Reply:
Good catch!

I changed it a bit, create a separate directory based on "inventory_host" - this way we should be able to do selective cleanup as well.